src/features/overlay/slice.ts

```diff
@@ -0,0 +1,53 @@
+/********************************************************************************
+ * Copyright (c) 2024 BMW Group AG
```

Comment:
you don't need to add `BMW Group AG`:) `Contributors to the Eclipse Foundation` is sufficient

Reply:
Noted.
Updated this file.